templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/__path__/__name__.component.ts

```diff
@@ -0,0 +1,314 @@
+import {
+    Component,
+    AfterViewInit, ElementRef, OnDestroy, OnInit, ViewChild, NgZone
+} from '@angular/core';
+import {
+    IgxButtonGroupComponent, IgxSliderComponent,
+    SortingDirection,
+    IgxTreeGridComponent,
+    PositionSettings,
+    OverlaySettings,
+    ConnectedPositioningStrategy,
+    AbsoluteScrollStrategy,
+    VerticalAlignment,
+    HorizontalAlignment
+} from 'igniteui-angular';
+import { timer } from 'rxjs';
+import { debounce } from 'rxjs/operators';
+import { LocalDataService } from './localData.service';
+import { ITreeGridAggregation } from './tree-grid-grouping.pipe';
+
+@Component({
+    providers: [LocalDataService],
+    selector: 'app-$(filePrefix)',
+	templateUrl: './$(filePrefix).component.html',
```

Comment:
This and the next line should be `Space` instead of `Tab` (to be consistent with the above lines)